tests/console/docker_image.pm

```diff
@@ -69,45 +45,45 @@ sub run {
         systemctl('restart docker');
     }
 
-    assert_script_run('curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64', 240);
-    assert_script_run('chmod +x container-diff-linux-amd64 && sudo mv container-diff-linux-amd64 /usr/local/bin/container-diff');
+    if (check_var("ARCH", "x86_64")) {
+        assert_script_run('curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64', 240);
+        assert_script_run('chmod +x container-diff-linux-amd64 && sudo mv container-diff-linux-amd64 /usr/local/bin/container-diff');
+    }
 
-    for my $i (0 .. $#image_names) {
+    for my $i (0 .. $#$image_names) {
         # Load the image
-        assert_script_run("docker pull $image_names[$i]", 900);
+        assert_script_run("docker pull $image_names->[$i]", 900);
```

Comment:
I know it is strange but our internal network can be very slow, please see the [screenshot](http://pdostal-server.suse.cz/tests/980#step/docker_image/28) and increase the timeout.

Reply:
By how much? It was 900 even before this PR, so it seems to be enough for most cases.

Reply:
Put it `1000` please 😞 I honestly don't know what is going on but you see the screenshot.

Reply:
I don't have a problems with longer timeouts. I would even suggest something higher but even, e.g. 1200. But please keep in mind that the sum of all internal timeouts should not exceed MAX_JOB_TIME which by default is 7200. You can set a higher MAX_JOB_TIME in all scenarios though

Reply:
Bumped to 1000 now.